integration-tests/bats/sql-local-remote.bats

```diff
@@ -439,3 +451,48 @@ get_staged_tables() {
     [ "$status" -eq 1 ]
     [[ "$output" =~ "When a password is provided, a user must also be provided" ]] || false
 }
+
+@test "sql-local-remote: verify dolt conflicts resolve behavior" {
+  skip "This test relies on dolt commit being migrated"
+
+  start_sql_server defaultDB
+  cd defaultDB
+
+  basic_conflict
+  dolt --user dolt checkout main
+  run dolt --user dolt sql -q "select * from t"
+  [ $status -eq 0 ]
+  [[ $output =~ "main" ]] || false
+
+  run dolt --user dolt merge other
```

Comment:
You are going to need to drop all of the `--user` options now that authentication change is in.

Reply:
Removing.